packages/core/src/components/HelpDrawer/HelpDrawer.jsx

```diff
@@ -0,0 +1,79 @@
+import Button from '../Button/Button';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export class HelpDrawer extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.titleRef = null;
+  }
+
+  componentDidMount() {
+    if (this.titleRef) this.titleRef.focus();
+  }
+
+  render() {
+    const {
+      ariaLabel,
+      title,
+      children,
+      onCloseClick,
+      footerBody,
+      footerTitle
+    } = this.props;
+    /* eslint-disable jsx-a11y/no-noninteractive-tabindex, react/no-danger */
+    return (
+      <div className="ds-c-help-drawer">
+        <div className="ds-c-help-drawer__header">
+          {/* The nested div below might seem redundant, but we need a
+            * separation between our sticky header, and the flex container
+            * so things display as expected when the body content overflows
+            */}
+          <div className="ds-u-fill--gray-lightest ds-u-padding--2 ds-u-display--flex ds-u-align-items--start">
+            <h3
+              ref={el => (this.titleRef = el)}
+              tabIndex="0"
+              className="ds-u-text--lead ds-u-margin-y--0 ds-u-margin-right--2"
+            >
+              {title}
+            </h3>
+            <Button
+              aria-label={ariaLabel}
+              className="ds-u-margin-left--auto"
+              size="small"
+              onClick={onCloseClick}
+              variation="secondary"
+            >
+              Close
+            </Button>
+          </div>
+        </div>
+        <div className="ds-c-help-drawer__body ds-u-md-font-size--small ds-u-lg-font-size--base ds-u-padding--2">
+          {children}
+        </div>
+        <div className="ds-c-help-drawer__footer ds-u-fill--primary-alt-lightest ds-u-md-font-size--small ds-u-lg-font-size--base ds-u-padding--2">
+          <h4 className="ds-text ds-u-margin--0">{footerTitle}</h4>
+          {/* dangerouslySetInnerHTML since footerBody can include HTML tags */}
+          <p
+            className="ds-text ds-u-margin--0"
+            dangerouslySetInnerHTML={{ __html: footerBody }}
+          />
```

Comment:
What if we leave the danger up to the developer who uses the component? That is, we just expect a renderable node, and if the developer wants to render a string, they can turn it into html themselves. So we'd have:
```jsx
<div className="ds-text ds-u-margin--0">
  {footerBody}
</div>
```
and they could pass stuff like
```jsx
<HelpDrawer footerBody="just some text" />
<HelpDrawer footerBody={<span>some <em>special</em> stuff </span>} />
<HelpDrawer footerBody={<span dangerouslySetInnerHTML={{ __html: stuff }} />} />
```